fastlane/Fastfile

```diff
@@ -704,6 +735,10 @@ def screenshots_scheme
   "SimplenoteScreenshots"
 end
 
+def ui_tests_scheme
+  "SimplenoteUITests"
+end
```

Comment:
Pretty suer I wrote some of these `def name "value" end` constructs 🤔 .

Since then, I learned of a better way of doing it: using Ruby constants.

```ruby
UI_TESTS_SCHEME = "SimplenoteUITests"
```

The value of using a constant is two-fold. First, it's appropriate because this value won't change in the context of the lane execution. Second, it's shorter than using a method (and possibly more performant, although I'm not sure what Ruby does under the hood and it really doesn't matter in our use case).

Worth pointing out that using a variable (`ui_tests_scheme = "..."`) wouldn't work because the lane wouldn't be able to find it in its local scope, while the constant, being a constant, can be found.

Reply:
Done! Thank you 🙇